imsv-docs-docusaurus/openapi/endpoints/login/models/create-challenge-request.yaml

```
@@ -16,6 +16,6 @@ properties:
       - polygon-mainnet
       - polygon-amoy
   address:
-    description: The web3 address to which the challenge is directed.
+    description: The EOA address to which the challenge is directed.
```

This was better as it was. EOA is evm-specific. 
```suggestion
    description: The web3 address to which the challenge is directed.
```